blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+
+Remote work offers the opportunity to rethink work from first principles and ask ourselves: What does life look like when our jobs work for us, rather than the other way around? When you don’t have to commute, when no one is watching you work, when code shipped at 2 am is as valuable as code shipped at 10 am, what can your life become? Maybe your day will become nonlinear; maybe you'll split your workday in two; or maybe you'll work 4 days instead of 5.
+
+This is [r/battlestations](https://www.reddit.com/r/battlestations/) by way of Google Calendar: When your calendar is a creative space, what will you make?
+
+## Loïc Guychard, engineering manager, has a four-day work week
+
+[Loïc Guychard](https://about.sourcegraph.com/handbook/company/team#lo%C3%AFc-guychard), engineering manager at Sourcegraph, compares work time to closet space: “If you have it, you’re going to use it. Whether you use it for things that matter, that’s a different story.”
+
+To compress his closet space, Loïc only works four days a week. He works from 10 am to 7 pm and takes Fridays off. The four-day work week constrains him in a creative way, forcing him to make better decisions with his time.
+
+> “Time-spent-at-desk is the worst measure of efficiency in software engineering.” - Loïc Guychard, engineering manager
+
+He's able to make better decisions partially because his longer weekends give him more energy. “You get a very good reset out of a three-day weekend,” he says. Plus, “when I do get to my holiday, I’m not crawling like I used to.”
+
+![4 day work week calendar](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/4%20day.png)
+
+Loïc extends his flexibility to his reports, arguing that “Input hours are a bad metric for any sort of efficiency.” According to him, “Time-spent-at-desk is the worst measure of efficiency in software engineering. I literally don’t care what hours my reports are working. I just care about output.”
+
+For Loïc, the goal is to maximize output and minimize synchronous work. It’s up to the team to set a “minimum viable guaranteed availability”–outside of that window, he says, work should be entirely flexible.
+
+Loïc believes this so strongly that he’d refuse an offer from a company based on flexibility alone. I asked him to consider an inflexible dream job but he disagreed with the premise: “I doubt that it would be my dream job if it’s completely inflexible.”
+
+Flexibility isn’t just a benefit to be tallied up against other benefits–it says something about your company. ”In our industry,” Guychard argues, ”if you’re inflexible, you’re ignoring how we actually work and how software actually gets produced. Software is not something that gets produced on an inflexible schedule.”
```

Comment:
@lguychard The calendar post you contributed to is almost ready to go live. Would you mind taking a quick glance at your section and letting me know if it's accurate?

Reply:
@nickmyyz Documenting from our chat on Slack – since @lguychard is OOO we will go ahead, Loïc if you have anything you'd like to change after publishing just let us know.